src/components/RightSidebar/RightSidebar.vue

```diff
@@ -260,29 +261,14 @@ export default {
 	display: flex;
 }
 
-/**
- * Replace padding with margin to make the participant list scroll
- * properly behind the top field
+/*
+ * The field will fully overlap the top of the sidebar content so
+ * that elements will scroll behind it
  */
-#tab-participants {
-	margin-top: 10px;
-	padding-top: 0;
-	outline: none; /* remove the weird border that appears on focus */
-}
-
-/* Force scroll bars in tabs content instead of in whole sidebar. */
-::v-deep .app-sidebar-tabs {
-	height: calc(100% - 80px);
-
-	&__content {
-		overflow: hidden;
-
-		section {
-			height: 100%;
-
-			overflow-y: auto;
-		}
-	}
+::v-deep .app-navigation-search {
```

Comment:
You most likely don't need v-deep as the root component SearchBox is used in the ParticipantTab component and will share the same vue-scope-id :)
https://github.com/nextcloud/spreed/blob/d0094567770ef799f039c6f57c1d33ca19f6f60e/src/components/RightSidebar/Participants/ParticipantsTab.vue#L275

You can move this there

Reply:
adjusted and squashed

at that point I forgot that I had direct access and that it wasn't an external component

Reply:
Yes, it gets confusing very fast :p